lib/gemfury/command/app.rb

```diff
@@ -117,13 +117,16 @@ def sharing
       me = client.account_info['username']
       collaborators = client.list_collaborators
       if collaborators.empty?
-        shell.say "You (#{me}) are the only collaborator", :green
+        shell.say %Q(You (#{me}) are the only collaborator\n), :green
       else
         shell.say %Q(\n*** Collaborators for "#{me}" ***\n), :green
-        usernames = [me] + collaborators.map { |c| c['username'] }
-        shell.say usernames.join("\n")
+
+        va = [ %w{ username permission } ]
+        va << [ me, 'owner' ]
```

Comment:
I think one is only relevant for Personal ("user" type) accounts. For Orgs, the API includes the viewing user.